src/Stat.php

```diff
@@ -573,29 +568,33 @@ public static function showTable($itemtype, $type, $date1, $date2, $start, array
             $headers[] = __('Average time to closure');
         } else {
             if ($itemtype === Ticket::class) {
-                $html_output .= $output::showHeaderItem(__s('Take into account'), $header_num);
+                $html_output .= $output::showHeaderItem(__s('Take into account'), $header_num); //@phpstan-ignore staticMethod.notFound (see $is_html_output condition)
```

Comment:
I do not understand why phpstan is complaining here, but not a few lines up, nor in other methods using the same code... Annotations to ignore seems the less worse fix for that specific case.

I guess a better fix would be to rewrite deeper that class to separate html/non html renderings; but this would be out of the scope of the current fix.